feathr_project/feathrcli/data/feathr_user_workspace/nyc_driver_demo.ipynb

```diff
@@ -86,9 +86,429 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 8,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Requirement already satisfied: feathr in /Users/hnlin/IdeaProjects/feathr_sample/feathr_project (0.4.0)\n",
+      "Collecting azure-cli\n",
+      "  Downloading azure_cli-2.36.0-py3-none-any.whl (2.4 MB)\n",
+      "     |████████████████████████████████| 2.4 MB 9.0 MB/s            \n",
+      "\u001b[?25hRequirement already satisfied: pandavro in /Users/hnlin/IdeaProjects/feathr_sample/sample_feathr/lib/python3.9/site-packages (1.6.0)\n",
```

Comment:
Is this stuff with your personal username and file paths supposed to be in this diff?